### ISSUE TITLE
[BUG] New version v0.5.0 ignores `setup.cfg`?

### ISSUE THREAD

**Issue by dominiquesydow** [s-weigand/flake8-nb#249]
Thanks a lot for this tool! We use it across several of our projects in the [@volkamerlab](https://github.com/volkamerlab) and [@wolberlab](https://github.com/wolberlab) organizations.

**Describe the bug**

Is it possible that `flake8-nb` configurations from `setup.cfg` are ignored since the version update from v0.4.0 to v0.5.0? 
Do I have to make any updates to our procedure as described below?

**To Reproduce**

In the [CI](https://github.com/wolberlab/dynophores/blob/16b924e266b9a1a4e20edc5d2d0a43d40c225dea/.github/workflows/CI.yaml#L137-L141) for our `dynophores` package, we test notebooks with
```
flake8-nb --config setup.cfg docs/tutorials/*.ipynb
```

The `flake8-nb` configuration looks like this in [`setup.cfg`](https://github.com/wolberlab/dynophores/blob/16b924e266b9a1a4e20edc5d2d0a43d40c225dea/setup.cfg#L30-L34):
```
[flake8_nb]
max-line-length = 99
ignore = E402,E703,W503
exclude = 
    docs/.ipynb_checkpoints
```

[Last week](https://github.com/wolberlab/dynophores/runs/7740038147?check_suite_focus=true#step:8:1), the CI passed; [this week](https://github.com/wolberlab/dynophores/runs/7853890941?check_suite_focus=true#step:8:1), we see the following output:
```
$ flake8-nb --config setup.cfg docs/tutorials/*.ipynb
docs/tutorials/explore_view3d.ipynb#In[2]:1:1: E402 module level import not at top of file
...
docs/tutorials/explore_view3d.ipynb#In[8]:1:57: E703 statement ends with a semicolon
...
```

**Expected behavior**
`flake8-nb` ignores E402 and E703 errors as defined in the config file.

**Desktop (please complete the following information):**
Reproduced on macOS 12.3 and GitHub Actions server

**Comment by s-weigand** [comment]
Hi @dominiquesydow thanks for raising this issue!

[It looks like](https://github.com/PyCQA/flake8/blob/e249dc47dfd314c34bbd0946c3fad970a8b15d6c/src/flake8/options/config.py) the new version of `flake8` (`>=5.0.0`) uses hardcoded `"flake8"` instead of the application name to find valid config files and sections.

Since users might want to have different rules for notebooks and python files, it's time for another hack, to fix this regression and restore the `flake8-nb<0.5.0` behaviour 😅

**Comment by s-weigand** [comment]
@all-contributors please add @dominiquesydow  for bug

**Comment by s-weigand** [comment]
@dominiquesydow I just released version `0.5.1` which should fix the issue (in a very hacky way 😅)
Feel free to reopen this issue the problem persists

**Comment by dominiquesydow** [comment]
@s-weigand, brilliant! Thanks so much for this quick fix - and for adding me to the contributor list for the bug report, much appreciated!
